content/en/platform/corda/5.2/deploying-operating/deployment/deploying/_index.md

```diff
@@ -414,108 +411,399 @@ The following configuration is required for the Corda databases:
 * [Cluster Database](#cluster-database)
 * [State Manager Databases](#state-manager-databases)
 
+Corda supports using multiple databases in isolation to spread the load and improve horizontal scalability.
+All databases must be defined only once within the deployment configuration and any dependant component can simply reference the database by `id`. This greatly simplifies maintenance and reduces redundant information.
+
+{{< note >}}
+A database with an `id` set as `default` is always mandatory. Corda uses this database if nothing else is configured.
```

Comment:
So, if you specify a different database in all of the necessary places, is it really mandatory?

Reply:
It is, yes:
```
        "databases": {
            "type": "array",
            "minItems": 1,
            "title": "List of databases used by the platform",
            "items": {
                "type": "object",
                "allOf": [
                    {
                        "$ref": "#/$defs/databaseTemplate"
                    }
                ],
                "required": [
                    "name",
                    "host",
                    "port",
                    "type"
                ]
            },
            "$comment": "At least one 'default' database must be defined",
            "contains": {
                "type": "object",
                "properties": {
                    "id": {
                        "type": "string",
                        "const": "default"
                    }
                }
            }
        },
```